core/gethwrappers/shared/generated/link_token/link_token.go

```diff
@@ -1932,11 +1932,11 @@ func (LinkTokenOwnershipTransferred) Topic() common.Hash {
 }
 
 func (LinkTokenTransfer) Topic() common.Hash {
-	return common.HexToHash("0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")
+	return common.HexToHash("0xe19260aff97b920c7df27010903aeb9c8d2be5d310a2c67824cf3f15396e4c16")
```

Comment:
What causes this diff? Can we undo that?

Reply:
auto generated by the CI

Reply:
I know that, but I'd like it to stay the same. Why would the hash change was my question, did something in this onchain logic change?

Reply:
It shouldn't have. I will investigate.